### PR TITLE
add consul

### PR DESCRIPTION
**Pull request by michaelamattes**

### CODE REVIEW

**Comment by rndmh3ro** [comment]
Nomad and Consul are both distributed as binaries, too.
Nomad gets currently installed with pip, I think we install the wrong nomad here (https://pypi.org/project/nomad/).

We should use either rpm or the binaries for both.

**Comment by michaelamattes** [comment]
Lets install over rpm for booth.

File: Dockerfile.template [hunk 1]
```diff
@@ -69,7 +70,17 @@ RUN if [ -n "${DOCKER_VERSION}" ] && [ "${DOCKER_VERSION}" != "latest" ]; then D
 RUN if [ -n "${NOMAD_VERSION}" ] && [ "${NOMAD_VERSION}" != "latest" ]; then NOMAD="NOMAD==${NOMAD_VERSION}";fi && \
```
Comment (rndmh3ro) [inline Dockerfile.template:70]: This has to be fixed now, too.

Comment (michaelamattes) [reply Dockerfile.template]: fixed

**Review by rndmh3ro — APPROVED** [review]

**Pull request merged**